clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.01
+  memoryConstraint: 9437184 #9 * (1024 * 1024)
+fluentd-gcp:
+  cpuConstraint: 1
+  memoryConstraint: 220200960 #210 * (1024 * 1024)
+heapster:
+  cpuConstraint: 0.1
+  nemoryConstraint: 340787200 #325 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 1.7
+  memoryConstraint: 975175680 #930 * (1024 * 1024)
+kube-controller-manager:
+  cpuConstraint: 0.9
```

Comment:
Why this is higher than orginally:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/density.go#L178

I don't think we observed issues with that...

Reply:
It rarely happens that cpu usage is above 0.8, but it happens. I'm aware that this value is greater than in original test, but I don't know the reason why yet.